Services/CO.CDP.DataSharing.WebApi/Model/Form.cs

```diff
@@ -36,8 +51,8 @@ public record Form
     public required string ShareCode { get; init; }
 
     /// <example>0</example>
```

Comment:
Examples should be updated. I think if you remove it alltogether, it will be automatically generated from the enum, but you'll need to confirm.